src/ontology/fbbt-edit.obo

```diff
@@ -125490,6 +125490,231 @@ intersection_of: part_of FBbt:00005893 ! embryonic/larval peripheral nervous sys
+creation_date: 2021-10-01T10:04:23Z
+
+[Term]
+id: FBbt:00058273
+name: oviduct epithelial cell
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00000124 ! epithelial cell
+intersection_of: part_of FBbt:00058272 ! oviduct epithelium
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-10-01T10:09:16Z
+
+[Term]
+id: FBbt:00058274
+name: intestinal stem cell of the adult midgut-hindgut boundary
+def: "An intestinal stem cell at the posterior end of the adult midgut, near the boundary with the hindgut. These cells are not part of the midgut-hindgut hybrid zone, but are located within 30 um of it and may re-populate it upon injury. They express frizzled3 (Sawyer et al., 2017)." [FlyBase:FBrf0237229]
+subset: cur
+synonym: "OB-ISC" EXACT [FlyBase:FBrf0237229]
+synonym: "organ-boundary intestinal stem cell" EXACT [FlyBase:FBrf0237229]
+is_a: FBbt:00110208 ! intestinal stem cell of posterior adult midgut epithelium
+relationship: expresses http://flybase.org/reports/FBgn0027343 ! fz3
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-10-01T10:48:05Z
+
+[Term]
+id: FBbt:00058275
+name: epithelial cell of the adult midgut-hindgut hybrid zone
+def: "Cell of the adult gut epithelium at the boundary between the midgut and the hindgut. There are approximately 125 of such cells in mature adults (day 4), down from 300 during pupation. They express wingless, along with markers characteristic of both the midgut and the hindgut (Sawyer et al., 2017)." [FlyBase:FBrf0237229]
+subset: cur
+intersection_of: FBbt:00000124 ! epithelial cell
+intersection_of: part_of FBbt:00047207 ! adult midgut-hindgut hybrid zone
+relationship: expresses http://flybase.org/reports/FBgn0284084 ! wg
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-10-01T11:13:46Z
+
+[Term]
+id: FBbt:00058276
+name: adult hindgut epithelial cell
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00000124 ! epithelial cell
+intersection_of: part_of FBbt:00047168 ! adult hindgut epithelium
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-10-01T11:26:09Z
+
+[Term]
+id: FBbt:00058277
+name: adult foregut epithelial cell
+def: "." [FBC:Autogenerated]
+subset: cur
+is_a: FBbt:00000124 ! epithelial cell
+intersection_of: FBbt:00000124 ! epithelial cell
+intersection_of: part_of FBbt:00047164 ! adult foregut epithelium
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-10-01T11:29:28Z
+
+[Term]
+id: FBbt:00058278
+name: adult pyloric epithelial cell
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00000124 ! epithelial cell
+intersection_of: part_of FBbt:00007044 ! adult pyloric epithelium
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-10-01T11:31:10Z
+
+[Term]
+id: FBbt:00058279
+name: adult esophagus epithelium
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00047164 ! adult foregut epithelium
```

Comment:
as for crop, this should be a part_of relationship.
Here you can use 'gut epithelial section', because it is a subdivision of the long axis of the gut.